modules/viewing-the-logs.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
```

Comment:
While troubleshooting is a big part of roxctl, this feels like it could deserve its own section within the docs: there's troubleshooting collector section, I suppose we should have a section for central as well.

There we can put things like viewing logs, debug dumps + diagnostic bundles.

Thoughts?

Reply:
I think it's a good idea but maybe outside of the scope of this PR/issue - do we want to make a ticket for it and add it to our to-do list?

Reply:
Creating a new ticket for this activity SGTM. I think for now can we keep it the way it is since it falls outside the scope of work for this PR. 
WDYT @dhaus67 @kcarmichael08 ?